web/src/features/fba/components/map/FBAMap.tsx

```diff
@@ -63,7 +63,6 @@ export interface FBAMapProps {
 export const hfiSourceFactory = (url: string) => {
   return new XYZ({
     url: `${RASTER_SERVER_BASE_URL}/tile/{z}/{x}/{y}?path=${url}&source=hfi`,
-    interpolate: false,
```

Comment:
Why are you removing this?

Reply:
Couldn't tell ya. I've merged Conor's branch into mine, and that `interpolate` change has come from his branch.

Reply:
He hadn't run yarn update - and removed that to make it work. Can you please roll back the change?